test/swoosh/adapters/postmark_test.exs

```diff
@@ -31,14 +34,17 @@ defmodule Swoosh.Adapters.PostmarkTest do
 
   test "a sent email results in :ok", %{bypass: bypass, config: config, valid_email: email} do
     Bypass.expect bypass, fn conn ->
-      conn = parse(conn)
-      body_params = %{"Subject" => "Hello, Avengers!",
-                      "To" => "tony.stark@example.com",
-                      "From" => "steve.rogers@example.com",
-                      "HtmlBody" => "<h1>Hello</h1>"}
+      conn        = parse(conn)
+      body_params = %{
+        "Subject"  => "Hello, Avengers!",
+        "To"       => "tony.stark@example.com",
+        "From"     => "steve.rogers@example.com",
+        "HtmlBody" => "<h1>Hello</h1>"
+      }
+
       assert body_params == conn.body_params
-      assert "/email" == conn.request_path
-      assert "POST" == conn.method
+      assert "/email"    == conn.request_path
+      assert "POST"      == conn.method
```

Comment:
All the above: Style change, should be in a separate PR and across all adapters.